test/bootstrap.test.js

```diff
@@ -16,15 +16,15 @@ const sailsrc = {
   },
   connections: {
     testConnection: {
-      user: 'Susana',
-      port: 5432,
-      database: 'factory_associatior',
-      password: '',
-      dialect: 'postgres',
+      user: process.env.USER || 'Susana',
```

Comment:
For better testing, env variables were added. Defaults to your config. Please, Susa, chang your non-existent password. It is now committed en GitHub.